Physics2D.Diagnostics/Diagnostics/DebugView.cs

```diff
@@ -87,8 +90,17 @@ public DebugView(World world)
             AppendFlags(DebugViewFlags.Shape);
             AppendFlags(DebugViewFlags.Controllers);
             AppendFlags(DebugViewFlags.Joint);
+
+
+            _debuggableCategories = Category.All;
+
         }
 
+        public void AddDebugCategory(Category cat) => _debuggableCategories |= cat;
+        public void RemoveDebugCategory(Category cat) => _debuggableCategories &= ~cat;
```

Comment:
Let's write those with a block method body for consistency with the rest of the code.
Also somebody might turn on or off multiple categories so the name should be plural.
I say we model those after the Flags as well.
public void AppendCategoryFlags(Category flags)
public void RemoveCategoryFlags(Category flags)